scripts/subdetector_tests/g4_material_scan.py

```diff
@@ -99,5 +101,22 @@ def g4_material_scan(compact, start_point, direction, timeout=200):
         print('Cannot find {}'.format(args.compact))
         exit(-1)
 
-    df = g4_material_scan(args.compact, args.start_point, '0.2,0.01,1.0')
-    print(df)
+    start_point = np.array([float(v.strip()) for v in args.start_point.split(',')])
+    etas = np.linspace(args.eta_min, args.eta_max, args.eta_nbins)
+    mats_indices = odict()
+    # a data buffer for the X0 values of (eta, material), 50 should be large enough
+    data = np.zeros(shape=(len(etas), 50))
+    for i, eta in enumerate(etas):
+        if i % PROGRESS_STEP == 0:
+            print('Scanned {:d}/{:d} lines for {:.2f} < eta < {:.2f}'.format(i, len(etas), etas[0], etas[-1]))
+        direction = (np.cos(args.phi/180.*np.pi), np.sin(args.phi/180.*np.pi), np.sinh(eta))
```

Comment:
A feature request from @johnlajoie is to allow averaging over phi. If I were to take a stab at it, I'd go with e.g. `--phi 0:1:30` for an average between 0 and 30 at steps of 1 deg.

Reply:
> 
Good suggestion. I'll implement this.

Reply:
Implemented in a slightly different manner: `<min>[:<max>][:<step>]`. Max value and step (default 1) are both optional.
In this case:
`0` = [0]
`0:5` = [0, 1, 2, 3, 4, 5]
`0:5:5` = [0, 5]
The argument for `eta` is also using the same format now.